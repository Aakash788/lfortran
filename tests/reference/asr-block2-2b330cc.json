{
    "basename": "asr-block2-2b330cc",
    "cmd": "lfortran --indent --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/block2.f90",
    "infile_hash": "f226e55b99771ba4b72596b4730aff7cc4a90424918f2b71111d33f1",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-block2-2b330cc.stdout",
    "stdout_hash": "533bfea5889c5d38e1e66e9a582b3eb1b840f2f143b5debec375e501",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}