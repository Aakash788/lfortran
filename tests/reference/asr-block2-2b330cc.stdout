(TranslationUnit
    (SymbolTable
        1
        {
            hybrd:
                (Function
                    (SymbolTable
                        2
                        {
                            delta:
                                (Variable
                                    2
                                    delta
                                    []
                                    Local
                                    ()
                                    ()
                                    Default
                                    (Real 4 [])
                                    Source
                                    Public
                                    Required
                                    .false.
                                ),
                            fcn:
                                (Function
                                    (SymbolTable
                                        3
                                        {
                                            n:
                                                (Variable
                                                    3
                                                    n
                                                    []
                                                    In
                                                    ()
                                                    ()
                                                    Default
                                                    (Integer 4 [])
                                                    Source
                                                    Public
                                                    Required
                                                    .false.
                                                )
                                        })
                                    fcn
                                    (FunctionType
                                        [(Integer 4 [])]
                                        ()
                                        Source
                                        Interface
                                        ()
                                        .false.
                                        .false.
                                        .false.
                                        .false.
                                        .false.
                                        []
                                        []
                                        .false.
                                    )
                                    []
                                    [(Var 3 n)]
                                    []
                                    ()
                                    Public
                                    .false.
                                    .false.
                                ),
                            ratio:
                                (Variable
                                    2
                                    ratio
                                    []
                                    Local
                                    ()
                                    ()
                                    Default
                                    (Real 4 [])
                                    Source
                                    Public
                                    Required
                                    .false.
                                )
                        })
                    hybrd
                    (FunctionType
                        []
                        ()
                        Source
                        Implementation
                        ()
                        .false.
                        .false.
                        .false.
                        .false.
                        .false.
                        []
                        []
                        .false.
                    )
                    [abs@sabs]
                    []
                    [(BlockCall
                        -1
                        2 main
                    )]
                    ()
                    Public
                    .false.
                    .false.
                ),
            iso_c_binding:
                (IntrinsicModule lfortran_intrinsic_iso_c_binding),
            iso_fortran_env:
                (IntrinsicModule lfortran_intrinsic_iso_fortran_env),
            lfortran_intrinsic_builtin:
                (IntrinsicModule lfortran_intrinsic_builtin),
            lfortran_intrinsic_math:
                (IntrinsicModule lfortran_intrinsic_math)
        })
    []
)
